extensionsdk/Microsoft.Windows.DevHome.SDK/Microsoft.Windows.DevHome.SDK.idl

```diff
@@ -1380,7 +1380,50 @@ namespace Microsoft.Windows.DevHome.SDK
 
     // End of Dev Environments feature.
 
+    // Begin FileExplorerSourceControlIntegration APIs
 
+    // This interface is responsible for obtaining and relaying information on the repository
+    // from source control technology
+    [contract(Microsoft.Windows.DevHome.SDK.DevHomeContract, 5)]
+    interface ILocalRepository
+    {
+        // This function returns the requested information on a relative file path (under a
+        // tracked repository root folder) from the source control management technology.
+        // The requested information includes a predefined set of properties FE is aware of.
+        // These properties are System.VersionControl.Status, System.VersionControl.LastChangeDate, 
```

Comment:
This list is duplicated and has to stay in sync with the readme.

Maybe remove these properties and reference the readme, or vice-versa?

Reply:
Resolved in latest. Thanks!